bears/vcs/git/CISkipInspectBear.py

```diff
@@ -0,0 +1,72 @@
+import re
+
+from bears.vcs.git.GitCommitMetadataBear import GitCommitMetadataBear
+from coalib.bears.GlobalBear import GlobalBear
+from coalib.results.Result import Result
+
+
+class CISkipInspectBear(GlobalBear):
+    LANGUAGES = {'Git'}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+
+    CI_PROVIDERS_METADATA = {
+          'CircleCI': r'\[ci skip\]|\[skip ci\]',
```

Comment:
you dont need a dictionary.

All of the CI providers who support skipping, all support both.  Someone else did the analysis for you at https://github.com/srz-zumix/ci-skip ;-)

Only Appveyor has an extra method of skipping, and you already have special logic for AppVeyor, so you can treat it as a special case.